modules/caddyhttp/tracing/module_test.go

```diff
@@ -180,3 +180,11 @@ func TestTracing_ServeHTTP_Next_Error(t *testing.T) {
 		t.Errorf("expected error, got: %v", err)
 	}
 }
+
+func createRequestWithContext(method string, url string) *http.Request {
+	r, _ := http.NewRequest("GET", "/", nil)
```

Comment:
This doesn't look right. The variable `url` isn't being used, and the request always goes to `/`.

Reply:
yep you're right, copied it and didn't actually use the variables

Reply:
fixed! https://github.com/caddyserver/caddy/compare/36c145ec329095be75555b5e4b833743a831dc0b..48797e1dc3bc525895902a1b904741c3450cb29c